addons/hw_drivers/static/src/js/worker.js

```diff
@@ -41,9 +42,11 @@
 
                         // Here we execute the code coming from the pos, apparently $.parseHTML() executes scripts right away,
                         // Since we modify the dom afterwards, the script might not have any effect
+                        /* eslint-disable no-undef */
```

Comment:
`no-undef` is supposed to handle `undefined` by default: https://eslint.org/docs/rules/no-undef#typeof

Might be worth:

* seeing if using double quotes (as in their examples) works
* checking if flipping the test (`!(typeof ...)`) fixes the issue

If either / both works, apply them (not great but better than eslint-disable) and report the issue to the eslint maintainers, it'd be stupid that `typeof x === "undefined"` be special-cased but not `typeof x !== "undefined"`.